docker/compose/faucet/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Generated files from running docker containers
+waypoint.txt
```

Comment:
this looks like a commit other than https://github.com/diem/diem/pull/7610

Reply:
I can remove it if there's a concern. 7610 follows on top of some other cleanups in the space,